docs/source/tutorial_optimization.rst

```diff
@@ -0,0 +1,900 @@
+Tutorial: Model order reduction for PDE-constrained optimization problems
+=========================================================================
+
+.. include:: jupyter_init.txt
+
+A typical application of model order reduction for PDEs are
+PDE-constrained parameter optimization problems. These problems aim to
+find a local minimizer of an objective functional which underlies a PDE
+that needs to be solved for all evaluations. Mathematically speaking,
+for a physical domain :math:`\Omega \subset \mathbb{R}^d` and a
+parameter set :math:`\mathcal{P} \subset \mathbb{R}^P`, we want to find
+a local minimizer :math:`\mu \in \mathcal{P}` of
+
+.. math::
+
+   \min_{\mu \in \mathcal{P}} J(u_{\mu}, \mu),  \tag{P.a}
+
+where :math:`u_{\mu} \in V := H^1_0(\Omega)` is the solution of
+
+.. math::
+
+   \begin{equation} \label{eq:primal}
+   a_{\mu}(u_{\mu}, v) = f_{\mu}(v) \qquad \forall \, v \in V \tag{P.b}.
+   \end{equation}
+
+The equation :math:`\eqref{eq:primal}` is called the primal
+equation and can be arbitrarily complex. MOR methods in the context of
+PDE-constrained optimization problems thus aim to find a surrogate model
+of :math:`\eqref{eq:primal}` to reduce the computational costs of
+an evaluation of :math:`J(u_{\mu}, \mu)`.
+
+Since :math:`u_{\mu}` is always related to :math:`\mu`, we can also
+simplify (P) by using the so-called reduced objective functional
+:math:`\mathcal{J}(\mu):= J(u_{\mu}, \mu)`. Note that 'reduced' is not
+referring to the order of the model. We get the reduced optimization
+problem: Find a local minimizer of
+
+.. math::
+
+    
+   \min_{\mu \in \mathcal{P}} \mathcal{J}(\mu).  \tag{$\hat{P}$}
+
+There exist plenty of different methods to solve (:math:`\hat{P}`) by
+using MOR methods. Some of them rely on an RB method with traditional
+offline/online splitting, which typically result in a very online
+efficient approach. Recent research also tackles overall efficiency by
+overcoming the expensive offline phase, which we will discuss further
+below.
+
+In this tutorial, we use a simple linear scalar valued objective functional
+and an elliptic primal equation to compare different approaches that solve
+(:math:`\hat{P}`). For a more advanced problem class, we refer to 
```

Comment:
I do not mention the reference here anymore due to the comment of @sdrave because I do not keep the referenced github project up to date with the pyMOR releases.